server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -548,6 +567,7 @@ const DefaultApiCounterIntervalMS = 60000;
 // 1 means 100%, using 2 just for safety for incorrect calculations and meaning this feature disabled
 const DefaultApiFailureRateTerminationThreshold = 2;
 const DefaultApiMinimumCountToEnableTermination = 30;
+const DefaultConsecutiveFailedTresholdForLowerTotalRequests = 3;
```

Comment:
Let's say our retry count at the mongo layer is 5. Would we want to consider making this new consecutiveFailedThreshold be at least 5 in that case, or, more generically, match the max retry count?

Maybe retry count for Mongo has nothing to do with this - in that case, please ignore my comment!

Reply:
No they are two different things I just like to use number 3 lol.

Retry count is retry for a particular db command, while this `DefaultConsecutiveFailedTresholdForLowerTotalRequests` (sorry for the bad naming), is to record how many consecutive pod failure rate that are more than threshold.